backend/video/views.py

```diff
@@ -78,12 +79,46 @@ def get_video(request):
 
     # Convert audio only to boolean
     is_audio_only = is_audio_only.lower() == "true"
-
     if url is None:
         return Response(
             {"error": "Video URL not provided in query params."},
             status=status.HTTP_400_BAD_REQUEST,
         )
+    
+    ## TEMP: Handle audio_only files separately for google drive links 
+    if "drive.google.com" in url and is_audio_only:
+
+        # Construct a direct download link from the google drive url 
+        # get the id from the drive link 
+        file_id = url.split("/")[-2]
+        url = f"https://drive.google.com/uc?export=download&confirm=yTib&id={file_id}"
+
+        # Get the video metadata
+        title = urllib.request.urlopen(urllib.request.Request(url)).info().get_filename()
+        direct_audio_url = url
+        
+        # Set duration to 0
+        duration = timedelta(seconds=0)
```

Comment:
Suggestion -- Can you please try this: https://stackoverflow.com/a/22116732/
I am guessing it might become an overhead if the audio is too big.

Can you please try using this 30min audio (30MB) to see how much time it takes:
https://drive.google.com/file/d/1XT_KnkN2zAbIHEfSc49VCnN8xQr9ZnHF

If it isn't too time-consuming, let us maybe do it this way.
Also, the above answer is for python2. We might have to modify a few things for py3.

Reply:
If this is too much, we can just put a TODO here saying that we need to come-up with a proper solution for this.

Reply:
The time taken is not very high. The 30 minute video duration was calculated in 5 seconds.